drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/type/DecisionTableXLSResourceType.java

```diff
@@ -17,28 +17,38 @@
 package org.drools.workbench.screens.dtablexls.client.type;
 
 import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
 
 import com.google.gwt.user.client.ui.Image;
 import com.google.gwt.user.client.ui.IsWidget;
 import org.drools.workbench.screens.dtablexls.client.resources.DecisionTableXLSResources;
 import org.drools.workbench.screens.dtablexls.client.resources.i18n.DecisionTableXLSEditorConstants;
 import org.drools.workbench.screens.dtablexls.type.DecisionTableXLSResourceTypeDefinition;
+import org.guvnor.common.services.project.categories.Decision;
 import org.uberfire.client.workbench.type.ClientResourceType;
 
 @ApplicationScoped
 public class DecisionTableXLSResourceType
         extends DecisionTableXLSResourceTypeDefinition
         implements ClientResourceType {
 
+    public DecisionTableXLSResourceType() {
+    }
+
+    @Inject
+    public DecisionTableXLSResourceType(Decision category) {
```

Comment:
It'd be nice to be consistent with your use of ```final``` on method parameters (most occurrences in this PR are marked as ```final```)